dashboard/lib/projects_list.rb

```diff
@@ -109,8 +112,18 @@ def get_published_project_data(project)
         # Note that we are using the new :project_type field rather than extracting
         # it from :value. :project_type might not be present in unpublished projects.
         type: project[:project_type],
-        publishedAt: project[:published_at]
+        publishedAt: project[:published_at],
+        studentName: student_name(project),
```

Comment:
It might be worth inlining a comment that we explicitly don't want to send the student name over the wire. Though you did so in the PR summary (thanks!), I think it is worth emphasizing this privacy aspect inline.